github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```diff
@@ -95,3 +95,29 @@ periodics:
       resources:
         requests:
           memory: "200Mi"
+- name: periodic-kubevirtci-bump-centos-version
+  cron: "50 1 * * *"
```

Comment:
we dont need to run it every day,
it means there will be lots of providers that the only thing is changed is the centos
and we would take each and every change

i think once a month or two / or even just manually upon needs is better, as was until today